build/build.go

```diff
@@ -13,15 +13,15 @@ import (
 
 	"github.com/bblfsh/sdk/v3/assets/build"
 	"github.com/bblfsh/sdk/v3/driver/manifest"
+	"github.com/bblfsh/sdk/v3/internal/buildmanifest"
 	"github.com/bblfsh/sdk/v3/internal/docker"
-	"gopkg.in/yaml.v2"
 )
 
 const releaseManifest = ".manifest.release.toml"
 
 const (
 	dockerFileName = docker.FileName
-	manifestName   = "build.yml"
+	manifestName   = buildmanifest.Filename
```

Comment:
[optional nit] We seem to use `FileName` everywhere else, but `Filename` here. Separate packages, I realize, but it might help to be consistent.

Reply:
Unfortunately, the `Filename` is used in the original `manifest` package and other projects like `bblfshd` use that constant. So it cannot be renamed this way without breaking the build.

Reply:
Ah, well. It'll be fine as is then.